Source/JavaScriptCore/bytecode/BytecodeUseDef.cpp

```diff
@@ -352,6 +355,14 @@ void computeUsesForBytecodeIndexImpl(const JSInstruction* instruction, Checkpoin
         handleOpCallLike(instruction->as<OpCallIgnoreResult>());
         return;
 
+    case op_instanceof_fused: {
+        auto bytecode = instruction->as<OpInstanceofFused>();
+        useAt(OpInstanceofFused::getHasInstance, bytecode.m_constructor);
+        useAt(OpInstanceofFused::getPrototype, bytecode.m_value, bytecode.m_constructor, bytecode.m_dst);
+        useAt(OpInstanceofFused::instanceof, bytecode.m_value, bytecode.m_dst);
```

Comment:
m_dst should not be used for these checkpoints, right?

Reply:
I think we need to use `m_dst` for those checkpoints since we use it for storing the `hasInstance` and `prototype` gotten from `m_constructor`.

Since two `get_by_id`s are needed for getting the `hasInstance` and `prototype`, threes checkpoints are needed
```
op_instanceof_fused:  
<----- getHasInstance
... Get the hasInstance from m_constructor
<------ getPrototype
... Check custom case with the hasInstance
... Get the prototype from m_constructor
<------ instanceof
... Do m_value instanceof the prototype
```

In `DFGByteCodeParser.cpp`, we store the `hasInstance` and `prototype` to the `m_dst`
```
handleGetById(bytecode.m_dst, prediction, get(bytecode.m_constructor), CacheableIdentifier::createFromImmortalIdentifier(hasInstanceImpl), identifierNumber, getByStatus, type, nextCheckpoint());
itermediateIndex = progressToNextCheckpoint();
```

and in `handleOpInstanceofFusedCheckpoint` from `LLIntSlowPaths.cpp` we directly do
```
auto hasInstanceOrPrototype = getOperand(callFrame, bytecode.m_dst);
```

Thus, if we don't use `m_dst` then the `hasInstance` and `prototype` won't be propagated back to LLInt.